internal/polkavm/host_call/refine_functions.go

```diff
@@ -402,9 +394,9 @@ func Invoke(
 	resultInstr, resultGas, resultRegs, resultMem, hostCall, invokeErr := interpreter.Invoke(program, pvm.InstructionCounter, invokeGas, invokeRegs, pvm.Ram)
 
 	if bb, err := jam.Marshal([14]uint64(append([]uint64{uint64(resultGas)}, resultRegs[:]...))); err != nil {
-		return gas, withCode(regs, OOB), mem, ctxPair, nil // (OOB, ω8, μ, m)
+		return gas, regs, mem, ctxPair, ErrPanicf(err.Error()) // (panic, ω8, μ, m)
```

Comment:
☇ 😄

Reply:
I prefer text for this, because the icon is not clearly visible as comment